advice.c

```diff
@@ -164,13 +164,20 @@ int error_resolve_conflict(const char *me)
 		error(_("It is not possible to %s because you have unmerged files."),
 			me);
 
-	if (advice_resolve_conflict)
+	if (advice_resolve_conflict) {
 		/*
 		 * Message used both when 'git commit' fails and when
 		 * other commands doing a merge do.
 		 */
 		advise(_("Fix them up in the work tree, and then use 'git add/rm <file>'\n"
 			 "as appropriate to mark resolution and make a commit."));
+
+		if (!strcmp(me, "cherry-pick") || !strcmp(me, "revert"))
+			advise(_("In case of cherry-picking / reverting you can also use\n"
+				 "    git [cherry-pick | revert] --skip\n"
```

Comment:
Now that you have mentioned it, we should indeed also advise to `--continue` or `--abort`.

Reply:
Cool.  I'd try to word it similar to how existing commands already do it.  See `git rebase` for example.

Reply:
I'm also slightly confused why we single out cherry-pick and revert; rebase has a --skip option, so if this advice is relevant and useful, why isn't rebase included?  Is it just that rebase can't hit this part of advice.c or is there some other special reason?  (Was it oversight for rebase to not have already had a note here...or is it perhaps better to just leave it out in all cases?)

Reply:
@newren see [this](https://github.com/r1walz/git/pull/1#pullrequestreview-239249118) thread. First off, Yes, since rebase can't reach these areas, we prefer to only include advice for cherry-pick/revert but I also think that including "advice" here is probably overkill and should be avoided.